gtdynamics/universal_robot/sdf.cpp

```diff
@@ -106,13 +105,34 @@ LinkSharedPtr LinkFromSdf(uint8_t id, const sdf::Link &sdf_link) {
   const auto &I = sdf_link.Inertial().Moi();
   inertia << I(0, 0), I(0, 1), I(0, 2), I(1, 0), I(1, 1), I(1, 2), I(2, 0),
       I(2, 1), I(2, 2);
-  const auto wTl = Pose3FromIgnition(sdf_link.Pose());
+
+  /// Call SemanticPose::Resolve so the pose is resolved to the correct frame
+  /// http://sdformat.org/tutorials?tut=pose_frame_semantics&ver=1.7&cat=specification&
+  // Get non-const Pose3d
+  auto jTl = sdf_link.RawPose();
```

Comment:
maybe *not* use the naming convention now but `raw_pose` as this pose will be changed in place (very bad practice if you ask me).

Reply:
Agreed, but I am not going to get into a flamewar with the SDFormat folks on what style of programming to use.